app/controllers/assertion_moderations_controller.rb

```diff
@@ -14,8 +14,11 @@ def moderation_params
     if params[:variant].present?
       params[:variant] = params[:variant][:name]
     end
+    if params[:nccn_guideline].present?
+      params[:nccn_guideline] = params[:nccn_guideline][:name]
+    end
 
-    params.permit(:summary, :description, :gene, :variant, :disease, :nccn_guideline, :nccn_guideline_version, :evidence_type, :evidence_direction, :amp_level, :clinical_significance, :drug_interaction_type, :fda_regulatory_approval, :fda_companion_test, :variant_origin)
+    params.permit(:summary, :description, :gene, :variant, :disease, :nccn_guideline_version, :evidence_type, :evidence_direction, :amp_level, :clinical_significance, :drug_interaction_type, :fda_regulatory_approval, :fda_companion_test, :variant_origin)
```

Comment:
I'd have to double check but with this as a new entity and not a field directly on Assertions anymore, do we need to update the suggested change logic to handle this appropriately? (more like a drug or source)

Reply:
This is not a many-to-many relationship so this should still work; it's more like diseases in that regard. (I don't think I was supposed to remove it from this list)

Reply:
Ah yeah, assuming this works as written, i think its good to go then.